### DISCUSSION TITLE
BattleBoxes Multiplayer Release 1.2.0

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#104] [Releases]
| Version | Changes                      |
| ------- | ---------------------------- |
| 1.1.1 | <ul><li>Added ingame debug console (Add your name to ops.json)</li><li>Added Content Security Policy</li><li>Added MeadowGuarder ad</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/68" target="_blank">Fixed non-ingame players being displayed (Issue #68)</a></li><li>Fixed username and password input alignment</li><li>Fixed more minibugs and nuisances</li></ul> |
| 1.1.2 | <ul><li>Hid copyright disclaimer and github link during games</li><li>Added chat</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/69" target="_blank">Fixed non-ingame players displaying after leaving game (Issue #69)</a></li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/72" target="_blank">Fixed crashes after leaving and disconnecting too quickly (Issue #72)</a></li><li>Added support for Heroku deploy</li><li>Moved Meadow Guarder ad to "Meadow Guarder" section</li><li>Updated server logging</li><li>Fixed getting stuck in collisions</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/73" target="_blank">Fixed PLAYER_LIST not clearing after rounds and resulting in "ghost players" (Issue #73)</li></ul> |
| 1.2.0 | <ul><li>Added lootboxes with placeholder images (can only be spawned via commands)</li><li>Implemented local settings saving via cookies</li><li>Removed right click menu</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/85" target="_blank">Fixed non-updating statistics (Issue #85)</a></li><li>Improved chat (and added anti-spam and anti-advertising)</li><li>Synced bullets on client with server</li><li>Optimized and rewrote some client-side code</li><li><a href="https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/issues/80" target="_blank">Removed random exclamation mark (Issue #80)</a></li><li>Mass glitch-fix</li></ul> |

<hr /><em>This discussion was created from the release <a href='https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/releases/tag/v1.2.0'>BattleBoxes Multiplayer Release 1.2.0</a>.</em>

**Comment by SampleProvider** [comment]
wooooo
meadow guarder ad
